src/Handler/CurlFactory.php

```diff
@@ -375,6 +375,14 @@ private function applyHandlerOptions(EasyHandle $easy, array &$conf)
             $conf[CURLOPT_TIMEOUT_MS] = $options['timeout'] * 1000;
         }
 
+        if (isset($options['force_ipv4'])) {
+            if (defined('CURLOPT_IPRESOLVE') && defined('CURL_IPRESOLVE_V4')) {
```

Comment:
From which PHP version is this feature available? Is it PHP version dependent at all? For me it looks a little bit weird that we check the feature and then refer to the PHP version. I would rather check the PHP version (if that's still necessary) or say this curl feature is not available. WDYT?